transactions-v1/sell_item.cdc

```diff
@@ -0,0 +1,43 @@
+import FlowToken from 0x0ae53cb6e3f42a79
+import FungibleToken from "../contracts/utility/FungibleToken.cdc"
+import NonFungibleToken from "../contracts/utility/NonFungibleToken.cdc"
+import ExampleNFT from "../contracts/utility/ExampleNFT.cdc"
+import NFTStorefront from "../contracts/NFTStorefront.cdc"
+
```

Comment:
Can you add a few more comments to your transactions?

Most transactions should have:
* high level description at the top (using `///`)
* a few comments in the code to describe lines that aren't obvious. (using `//`)